cmd/internal/resolve/resolve.go

```diff
@@ -77,19 +77,28 @@ func AddTo(c commander.Interface) {
 
 			}
 
-			r := resolver.NewDefaultSatResolver(p, phonyCatalogSourceLister{}, log)
+			sp := resolver.SourceProviderFromRegistryClientProvider(p, logrus.New())
```

Comment:
If you reuse `log` here then it'll honor the --verbose flag:

```suggestion
			sp := resolver.SourceProviderFromRegistryClientProvider(p, log)
```